it/src/main/java/org/corfudb/universe/universe/vm/VmConfigFileUtil.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.universe.universe.vm;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Properties;
+
+public class VmConfigFileUtil {
```

Comment:
Comments. Even by reading the class name, I have no idea what this class does. Maybe ConfigPropertiesLoader?

Reply:
done (doc + renamed)